pkg/disruption/backend/sampler/remote.go

```diff
@@ -0,0 +1,501 @@
+								},
+								{
+									Name:  "API_DISRUPTION_ONLY",
+									Value: "true",
+								},
+								{
+									Name:  "LOAD_BALANCER_TYPE",
+									Value: "internal-lb",
+								},
+							},
+							Command: []string{
+								"openshift-tests",
+								"run-monitor",
+								"--artifact-dir",
+								disruptionDataPath,
+							},
+							VolumeMounts: []corev1.VolumeMount{
+								{
+									Name:      "artifacts",
+									MountPath: disruptionDataPath,
+								},
+							},
+						},
+						{
+							Name:  "service-network",
+							Image: "image-registry.openshift-image-registry.svc:5000/openshift/tests:latest",
+							Command: []string{
+								"openshift-tests",
+								"run-monitor",
+								"--artifact-dir",
+								disruptionDataPath,
+							},
+							Env: []corev1.EnvVar{
+								{
+									Name: "EXTRA_MESSAGE",
+									ValueFrom: &corev1.EnvVarSource{
+										FieldRef: &corev1.ObjectFieldSelector{
+											FieldPath: "spec.nodeName",
+										},
+									},
+								},
+								{
+									Name:  "API_DISRUPTION_ONLY",
+									Value: "true",
+								},
+								{
+									Name:  "LOAD_BALANCER_TYPE",
+									Value: "service-network",
+								},
+							},
+							VolumeMounts: []corev1.VolumeMount{
+								{
+									Name:      "artifacts",
+									MountPath: disruptionDataPath,
+								},
+							},
+							SecurityContext: &corev1.SecurityContext{
+								Privileged: &bTrue,
+							},
+						},
+					},
+					ServiceAccountName: serviceAccountName,
+					Tolerations: []corev1.Toleration{
+						{
+							Key:    "node-role.kubernetes.io/master",
+							Effect: corev1.TaintEffectNoSchedule,
+						},
+					},
+				},
+			},
+		},
+	}
+
+	return createDaemonset(ctx, clientset, create, obj)
+}
+
+func callRBACClusterAdmin(ctx context.Context, clientset *kubernetes.Clientset, create bool) error {
+	obj := &rbacv1.ClusterRoleBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      fmt.Sprintf("%s-admin", serviceAccountName),
+			Namespace: namespace,
+		},
+		RoleRef: rbacv1.RoleRef{
+			Kind: "ClusterRole",
+			Name: "cluster-admin",
+		},
+		Subjects: []rbacv1.Subject{
+			{
+				Kind:      rbacv1.ServiceAccountKind,
+				Name:      serviceAccountName,
+				Namespace: namespace,
+			},
+		},
+	}
+
+	client := clientset.RbacV1().ClusterRoleBindings()
+	var err error
+	if create {
+		_, err = client.Create(ctx, obj, metav1.CreateOptions{})
+	} else {
+		err = client.Delete(ctx, obj.Name, metav1.DeleteOptions{})
+	}
+
+	if apierrors.IsAlreadyExists(err) || apierrors.IsNotFound(err) {
+		return nil
+	}
+	return err
+}
+
+func callRBACHostaccess(ctx context.Context, clientset *kubernetes.Clientset, create bool) error {
+	obj := &rbacv1.ClusterRoleBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      fmt.Sprintf("%s-privileged", serviceAccountName),
+			Namespace: namespace,
+		},
+		RoleRef: rbacv1.RoleRef{
+			Kind: "ClusterRole",
+			Name: "system:openshift:scc:privileged",
```

Comment:
huh, I remain to be convinced.  you need hostnetwork in some cases, but all privileged?

Reply:
Seems `hostaccess` SCC is insufficient to write to `/var/log` on node. Not sure why, but `privileged` SCC allows it